docs/dns/domains.rst

```diff
@@ -257,6 +257,25 @@ and ``subname`` would just be ``_acme-challenge``.
 The above API request helps you answer this kind of question.
 
 
+Exporting a Domain as Zonefile
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To export domain data in zonefile format, send a ``GET`` request to the
+``zonefile`` endpoint of this domain, i.e. to ``/domains/{name}/zonefile``::
+
+    curl -X GET https://desec.io/api/v1/domains/{name}/zonefile \
+        --header "Authorization: Token {token}"
+
+Note that this will return all data for this domain in plain zonefile
+format without JSON formatting, e.g.::
+
+    ; Zonefile for example.com exported from desec.io at 2022-08-26 16:03:18.258961+00:00
+    example.com.	1234	IN	NS	ns1.example.com.
+    example.com.	1234	IN	NS	ns2.example.com.
+    example.com.	300	IN	SOA	get.desec.io. get.desec.io. 2022082602 86400 3600 2419200 3600
+
+DNSSEC-specific record types are not included.
```

Comment:
This should preferably go before the code listing, because in the visual rendering, this sentence is easy to miss. (Don't feel strongly.)

Reply:
fixed